### ISSUE TITLE
It appears like admin view order option upload attachment is not working

### ISSUE THREAD

**Issue by hayden-t** [opencart/opencart#12944]
**What version of OpenCart are you reporting this for?**
master

**Describe the bug**
it appears like the attachment name (not unique) is put into the order_options table rather than the code (which is)

its doing this because even though the code is stored in the cart table, when the order is confirmed and added to db orders table, it gets its content from the cart get products function which switch's the code for the name, thus only the name gets put in db orders and then later in admin cannot be used to get the download out again

here is where the switch happens
https://github.com/opencart/opencart/blob/abcd6b14afba3fb66530177fb19532d14bcb7990/upload/catalog/model/checkout/cart.php#L37

here is where this function above is called 
https://github.com/opencart/opencart/blob/abcd6b14afba3fb66530177fb19532d14bcb7990/upload/catalog/controller/checkout/confirm.php#L37

and here the data is sent to the model to be put in db on finish checkout
https://github.com/opencart/opencart/blob/abcd6b14afba3fb66530177fb19532d14bcb7990/upload/catalog/controller/checkout/confirm.php#L311

**Comment by hayden-t** [comment]
tldr: 81590ENTB.pdf is being saved in order_option rather than 226c403e6778a68f38d74289408426a9
